lib/ridgepole/delta.rb

```diff
@@ -328,7 +328,7 @@ def append_change_table_raw_options(table_name, raw_table_options, table_charset
     end
 
     def append_change_table_comment(table_name, table_comment, buf)
-      comment_literal = "COMMENT=#{ActiveRecord::Base.connection.quote(table_comment)}"
+      comment_literal = "COMMENT=#{ActiveRecord::Base.connection.quote(table_comment)}".gsub('\\') { '\\\\' }
```

Comment:
This is recommended way at `#gsub` doc

https://docs.ruby-lang.org/ja/latest/method/String/i/gsub.html#:~:text=%E3%81%93%E3%81%AE%E3%82%88%E3%81%86%E3%81%AA%E9%96%93%E9%81%95%E3%81%84%E3%82%92%E7%A2%BA%E5%AE%9F%E3%81%AB%E9%98%B2%E6%AD%A2%E3%81%97%E3%80%81%E3%82%B3%E3%83%BC%E3%83%89%E3%81%AE%E5%8F%AF%E8%AA%AD%E6%80%A7%E3%82%92%E4%B8%8A%E3%81%92%E3%82%8B%E3%81%AB%E3%81%AF%E3%80%81%20%5C%26%20%E3%82%84%20%5C1%20%E3%82%88%E3%82%8A%E3%82%82%E4%B8%8B%E8%A8%98%E3%81%AE%E3%82%88%E3%81%86%E3%81%AB%E3%83%96%E3%83%AD%E3%83%83%E3%82%AF%E4%BB%98%E3%81%8D%E5%BD%A2%E5%BC%8F%E3%81%AE%20gsub%20%E3%82%92%E4%BD%BF%E3%81%86%E3%81%B9%E3%81%8D%E3%81%A7%E3%81%99%E3%80%82

Reply:
Thank you 😃 

This is an issue with the `append_change_table_options method`, so I fixed `append_change_table_options` method. https://github.com/ridgepole/ridgepole/pull/469

Could you please rebase this pull request on the `2.0` branch and undo this fix?

```suggestion
      comment_literal = "COMMENT=#{ActiveRecord::Base.connection.quote(table_comment)}"
```

Reply:
Thanks, I learned a lot.

> Could you please rebase this pull request on the 2.0 branch and undo this fix?

Done at this commit. And changed title 👍 
https://github.com/ridgepole/ridgepole/pull/467/commits/719f1918b5309b0052ff2d49ceca624c01d13d29